src/playdate/system.nim

```diff
@@ -33,9 +36,41 @@ proc getSecondsSinceEpoch* (this: ptr PlaydateSys): tuple[seconds: uint, millise
 type PDCallbackFunction* = proc(): int {.raises: [].}
 var updateCallback: PDCallbackFunction = nil
 
+proc runCatchingTyped*[T](fun: () -> T, fatal: bool = false, messagePrefix: string=""): T =
+    try:
+        return fun()
+    except:
+        let exception = getCurrentException()
+        var message: string = ""
+        try:
+            message = &"{messagePrefix}: {getCurrentExceptionMsg()}\n{exception.getStackTrace()}"
+            # replace line number notation from (90) to :90, which is more common and can be picked up as source link
+            message = message.replace('(', ':')
```

Comment:
Apply this replacement to the stacktrace only

Reply:
...Or perhaps remove it completely.

Vscode terminal can pick up line numbers in brackets and IntelliJ can do it too when the "Awesome Console" plugin is installed
![Screenshot 2025-03-09 at 11 57 59](https://github.com/user-attachments/assets/0af4c8e4-4d74-4dda-b7c5-b8ce7c7babfd)

Reply:
I'm fine either way